### PR TITLE
Add build and links test

### PR DESCRIPTION
**Pull request by doktorbro**
This adds a continuous testing with [Proofer](https://github.com/gjtorikian/html-proofer). 

> Proofer is a set of tests to validate your HTML output. These tests check if your image references are legitimate, if they have alt tags, if your internal links are working, and so on.

You must [enable Travis](http://docs.travis-ci.com/user/getting-started/) to see it working.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -2,11 +2,12 @@
 title: Grid by Example
 email: me@rachelandrew.co.uk
 description: "My collection of simple examples of the CSS3 Grid Layout Module."
-baseurl: ""
 url: "http://gridbyexample.com"
 twitter_username: rachelandrew
 github_username:  rachelandrew
 
 # Build settings
+baseurl: ""
```
Comment (doktorbro) [inline _config.yml:10]: `baseurl` is a build setting.

**Comment by rachelandrew** [comment]
As I don't use these tools, I am not going to add complexity to my own environment by merging this. I end up working on a variety of machines and the idea of using Jekyll was simply so I could write the stuff quickly in markdown and drop it in. Adding a whole stack of tools defeats the object somewhat.

**Comment by doktorbro** [comment]
Travis would extend Github’s functionality, not your machines.

**Comment by rachelandrew** [comment]
Until I have chance to find out what something is and does I won't just include it I'm afraid. I'm old-school like that :)

**Pull request closed without merging**